addons/web_editor/static/src/js/editor/image_processing.js

```diff
@@ -334,7 +334,14 @@ async function applyModifications(img, dataOptions = {}) {
     ctx.fillRect(0, 0, result.width, result.height);
 
     // Quality
-    return result.toDataURL(mimetype, quality / 100);
+    const dataURL = result.toDataURL(mimetype, quality / 100);
+    // Every 4 bytes of base64 represent 3 bytes.
+    const newSize = dataURL.split(',')[1].length / 4 * 3;
```

Comment:
Maybe you could create a utility function to get the size from the dataURL because we already do that to display the current file size of images in the editor see `ImageHandlerOption`